### PR TITLE
Remove all files and export new package

### PR DESCRIPTION
**Pull request by nshahan**
Deprecate angular2_components library. 
Update README to alert users to the change.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 0.5.0-beta 
+
+This code is considered production quality, but depends on angular2: 
+^3.0.0-alpha+1. The alpha tag represents the evolving nature of the AngularDart 
+api, not code quality (3.0.0-beta+1 is used in production Google apps). 
+ 
+ * Rename library to angular_components.
```
Comment (kevmoo) [inline CHANGELOG.md]: Remove this

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+> NOTE: This package has been renamed to angular_components without the "2". 
+> To continue to receive the latest versions update your `pubspec.yaml` to 
+> depend on `angular_components: ^0.5.0-beta`. 
+
```
Comment (kevmoo) [inline README.md]: Put a link here to the new package URI on pub.dartlang.org.

And delete the rest of the README.

File: lib/angular2_components.dart [hunk 3]
```diff
@@ -2,69 +2,11 @@
...
-  NgModel,
-  PopupSourceDirective,
-  ReorderItemDirective,
-  ReorderListComponent,
-  ScorecardComponent,
-];
+// This package has been renamed to angular_components without the "2".
+// To continue to receive the latest versions update your pubspec.yaml to
+// depend on angular_components: ^0.5.0-beta.
+@deprecated
```
Comment (kevmoo) [inline lib/angular2_components.dart]:
```dart
@Deprecated("Update your dependency to use `angular_components` (without the "2") instead.")
```

File: pubspec.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 name: angular2_components
-version: 0.4.1-beta
+version: 0.5.0-beta
 description: >
   The official Material Design components for AngularDart. Used at Google
```
Comment (kevmoo) [inline pubspec.yaml]: Update the description, too. Use the content from the `@Deprecated` content

File: CHANGELOG.md (hunk 1 above)
Comment (nshahan) [inline CHANGELOG.md]: Do you want to see anything in the changelog about this?

Comment (kevmoo) [inline CHANGELOG.md]: Shoot. Messed up.

The rename should be at the TOP of the changelog. With **BOLD** text.

File: pubspec.yaml [hunk 5]
```diff
@@ -1,8 +1,7 @@
 name: angular2_components
 version: 0.5.0-beta
 description: >
-  The official Material Design components for AngularDart. Used at Google
-  in production apps.
+  Update your dependency to use `angular_components` (without the "2") instead.
```
Comment (kevmoo) [inline pubspec.yaml]: Prepend "RENAMED:"

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,7 @@
+## 0.5.0-beta 
+
+__Rename library to angular_components.__
```
Comment (kevmoo) [inline CHANGELOG.md]: Put a link here, too. This package now exports from link_to_angular_comp...use that instead...

File: README.md [hunk 7]
```diff
@@ -1,66 +1,5 @@
-## Preview Release
+> NOTE: This package has been renamed to angular_components without the "2".
```
Comment (kevmoo) [inline README.md]: silly – put 'without the "2"') in parens – it is an aside.

Comment (kevmoo) [inline README.md]: angular_compenents in `ticks`

**Comment by kevmoo** [comment]
...you realize this branch is completely broken until we publish the other package...

**Pull request merged**